packages/docs/docs/docs/zos_lib.md

```diff
@@ -139,20 +135,26 @@ And now, let's write our upgrading script in `index.js`:
 global.artifacts = artifacts;
 global.web3 = web3;
 
-const { Contracts, SimpleProject  } = require('zos-lib')
+const { Contracts, SimpleProject, ZWeb3 } = require('zos-lib')
+ZWeb3.initialize(web3.currentProvider)
+
 // Load the contract.
 const MyContractV0 = Contracts.getFromLocal('MyContractV0');
 const MyContractV1 = Contracts.getFromLocal('MyContractV1');
 
 async function main() {
+
   // Instantiate a project.
-  const project = new SimpleProject('MyProject', { from: web3.eth.accounts[0] });
+  const initializerAddress = (await web3.eth.getAccounts())[1];
```

Comment:
Just curious: why the change from account 0 to 1 here?

Reply:
With 0 I was getting this error:
```
VM Exception while processing transaction: revert Cannot call fallback function from the proxy admin
```

Reply:
@spalladino 

Re:
> One question: what do you think about keeping the version identifiers for dependencies, to avoid any problems in the future? So, instead of suggesting to install plainly `npm i truffle`, suggest `npm i truffle@5.0`; same for `zos-lib@2.2` (note that it does not include the patch version, so it installs the latest patch), etc.

I thought about it and my conclusion was that documentation for @next versions could not have to specify the dependency versions. This simplifies the guides for the reader, but asumes that all the latest versions will just work. We could not make that assumption and just fix the dependency versions. Which one do you like best?

Reply:
> With 0 I was getting this error:

My bad. That example is using SimpleProject, and not ProxyAdminProject, so it makes sense that it requires a different one!

> I thought about it and my conclusion was that documentation for @next versions could not have to specify the dependency versions. This simplifies the guides for the reader, but asumes that all the latest versions will just work.

Let's specify versions for all dependencies (such as truffle), so if they release a new breaking version, that does not break our guides. As for zos and zos-lib, agree on leaving them without version identifiers for simplicity.